### DISCUSSION TITLE
accepted but no github co-pilot

### DISCUSSION THREAD

**Discussion by MennoSaey** [community/community#141065] [GitHub Education]
### Hi everyone,

hello, my request for github education has been approved on oct 2 but i still havent received access t co-pilot

**Comment by davevad93** [comment]
Hi @MennoSaey ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @MennoSaey was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by MennoSaey** [reply]
where can i see this status label

**Reply by davevad93** [reply]
Hey @MennoSaey you can check your status [here](https://education.github.com/discount_requests/application) where it says "***You've already submitted X requests***".
